### PR TITLE
SPCTR-2333 Fix: Forms/Tabs/Table of contents - JS throws an error in console on blog/home page. 

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-2333

### Screenshots

- https://share.getcloudapp.com/d5uyKv1n

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Added to the card

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: assets/js/forms.js [hunk 1]
```diff
@@ -3,7 +3,9 @@ UAGBForms = { // eslint-disable-line no-undef
 	init( attr, id, post_id ) {
 
 		const scope = document.querySelector( id );
-
+		if( null === scope ){
```
Comment (RajkiranBagal) [inline assets/js/forms.js:6]: check ` ! scope `

File: assets/js/tabs.js [hunk 2]
```diff
@@ -1,6 +1,9 @@
 UAGBTabs = { // eslint-disable-line no-undef
 	init( $selector ) {
 		const tabsWrap = document.querySelector( $selector );
+		if( null === tabsWrap ){
```
Comment (RajkiranBagal) [inline assets/js/tabs.js:4]: check ! tabsWrap

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: assets/js/forms.js (hunk 1 above)
Comment (AbhijitNage123) [reply assets/js/forms.js]: Done

File: assets/js/table-of-contents.js [hunk 3]
```diff
@@ -228,6 +228,9 @@ UAGBTableOfContents = { // eslint-disable-line no-undef
 			return decodeURI( encodeURIComponent( parsedSlug ) );
 		};
 		const $thisScope = document.querySelector( id );
+		if( null === $thisScope ){
```
[the remark this answers is not in the document]
Comment (AbhijitNage123) [reply assets/js/table-of-contents.js]: Done

File: assets/js/tabs.js (hunk 2 above)
Comment (AbhijitNage123) [reply assets/js/tabs.js]: Done

**Pull request merged**